app/Http/Controllers/TodoController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Todos;
+use Illuminate\Http\Request;
+
+class TodoController extends Controller
+{
+    
+    public function all(Request $request){
+        $user = auth()->user();
+        return response()->json($user->todos);
+    }
```

Comment:
ovo `all` mi nista ne znaci, malo deskriptivnije nazovi metodu. Npr. `allUserTodos`

Reply:
Jos jedna stvar, napravi service todo i onda unutar njega vracas , pises logiku i ostalo. Controller je tu samo da bi pozvao metodu u servisu i to je to. Duznost Controller-a je da izvalidira requesst i da proslijedi validirane podatke servisu.

Reply:
Takodje kada pravis servis, svaki servis je ispracen sa svojim interface-om. Hipoteticki govoreci, sutra ce ti trebati drugacija implementacija od one koju budes imao, pa ces samim interface-om da apstraktujes sam poziv metode, kasnije samo mijenjas njegove implementacije. Tako da ces sada imati `TodoService`  interface i `TodoServiceImpl` konkretnu implementaciju tog interface-a (bice klasa).